cfme/utils/dockerbot/sel_container.py

```diff
@@ -33,7 +33,7 @@ def vnc_ready(addr, port):
 @click.option('--image', help='Chooses selenium container image',
               default=docker_conf.get('selff', 'cfmeqe/sel_ff_chrome'))
 @click.option('--vncviewer', help='Chooses VNC viewer command',
-              default=docker_conf.get('vncviewer', 'vinagre'))
+              default=docker_conf.get('vncviewer', 'vncviewer'))
```

Comment:
Optional:
Mike one thought came in mind like better to make `xdg-open` as default. maybe user vncviewer not installed on user system but our most of user use `vncviewer`